pkg/kubectl/cmd/get.go

```diff
@@ -223,6 +225,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			FilenameParam(enforceNamespace, &options.FilenameOptions).
 			SelectorParam(selector).
 			ExportParam(export).
+			RequestChunksOf(options.ChunkSize).
```

Comment:
why not `ChunkSize()`?

Reply:
I didn't want to expose that yet, Limit itself is only of partial utility.  I was focused on "i want to do what i did before, but in chunks".